.github/workflows/ci.yml

```diff
@@ -377,7 +404,7 @@ jobs:
       - name: check --feature-powerset --unstable
         run: cargo hack check --all --feature-powerset --depth 2 -Z avoid-dev-deps --keep-going
         env:
-          RUSTFLAGS: --cfg tokio_unstable -Dwarnings
+          RUSTFLAGS: --cfg tokio_unstable --cfg tokio_taskdump -Dwarnings
```

Comment:
Since this is the test designed to test various combinations of flags, I suggest we also try with just `--cfg tokio_unstable`.
```yml
      # Try with unstable feature flags
      - name: check --feature-powerset --unstable
        run: cargo hack check --all --feature-powerset --depth 2 -Z avoid-dev-deps --keep-going
        env:
          RUSTFLAGS: --cfg tokio_unstable -Dwarnings
      # Try with unstable and taskdump feature flags
      - name: check --feature-powerset --unstable --taskdump
        run: cargo hack check --all --feature-powerset --depth 2 -Z avoid-dev-deps --keep-going
        env:
          RUSTFLAGS: --cfg tokio_unstable --cfg tokio_taskdump -Dwarnings
```